src/scene_battle.cpp

```diff
@@ -94,6 +94,8 @@ void Scene_Battle::Start() {
 	Main_Data::game_enemyparty.reset(new Game_EnemyParty());
 	Main_Data::game_enemyparty->Setup(troop_id);
 
+	Main_Data::game_pictures->UpdateGraphics(true);
```

Comment:
Should we do this after `Game_Battle::Init()`? It's in `Game_Battle::Init()` when `Game_Battle::IsBattleRunning()` becomes true. 

Now since this method takes a bool and doesn't rely on that global anymore, it should be fine. However I have a minor concern that maybe in the future someone adds something in the call chain of `UpdateGraphics` which depends on `IsBattleRunning()`

TLDR: It doesn't matter, but it might be a bit safer to move it after `Game_Battle::Init()` just to be sure.